src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
```

Comment:
<img width="959" alt="image" src="https://user-images.githubusercontent.com/102411130/202660761-0c3e043e-d2da-476c-912e-909100dc5408.png">
checklist
don't use `tag` selectors for styles

Reply:
Referred to Tips & Hints: Don't use repeated styles.
ok, will fix it.